src/System.Text.Encoding.CodePages/src/System/Text/EncodingTable.Data.cs

```diff
@@ -1250,6 +1250,7 @@ internal static partial class EncodingTable
             28599, // iso-8859-9
             28603, // iso-8859-13
             28605, // iso-8859-15
+            29001, // x-europa
```

Comment:
Did you edit this file by hand? we have some tool which auto generate this file to ensure the right data is generated. we can share the tool with you. we have another open issue against me to open source this tool too but I didn't get chance yet to do it.